po/common/en.po

```diff
@@ -4827,6 +4827,19 @@ msgctxt "attribute_group_environment_name"
 msgid "Environment"
 msgstr "Environment"
 
+msgctxt "attribute_environmental_score_name"
+msgid "Eco-Score"
+msgstr "Eco-Score"
+
+msgctxt "attribute_environmental_score_setting_name"
+msgid "Low environmental impact (Eco-Score)"
+msgstr "Low environmental impact (Eco-Score)"
+
+msgctxt "attribute_environmental_score_setting_note"
+msgid "The Eco-Score is an environmental score from A to E which makes it easy to compare the impact of food products on the environment."
+msgstr "The Eco-Score is an environmental score from A to E which makes it easy to compare the impact of food products on the environment."
+
+# Note: the Eco-Score is renamed to Environmental Score, but we keep the ecoscore identifier as it is stored in clients
 msgctxt "attribute_ecoscore_name"
 msgid "Eco-Score"
 msgstr "Eco-Score"
```

Comment:
Is this ok to change that, if the attribute name does not change ?

Reply:
@alexgarel I add the new names, but I kept the old name as well, hopefully we can change the name later.